### PR TITLE
Jb49476

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

**Review by vigejolla — COMMENTED** [review]
One suggestion

File: sdk-setup/systemd/host_home.service [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Mount user home directory from host
+
+[Service]
+Type=oneshot
+WorkingDirectory=/
+EnvironmentFile=-/etc/mersdk.env.systemd
+EnvironmentFile=-/etc/login.defs.systemd
+ExecStartPre=/bin/mkdir -p /host_home
+ExecStart=/bin/mount -t vboxsf -o ro,uid=${UID_MERSDK},gid=${GID_MERSDK} home /host_home
```
[the remark this answers is not in the document]
Comment (martyone) [reply sdk-setup/systemd/host_home.service:10]: This is like with all the other services (not just the SDK-related) that must not be enabled under docker - all except those explicitly allowed are disabled from setup-buildengine. More proper would be to provide a package sdk-vm-docker which would mask all such services, but we are not there yet.

[the remark this answers is not in the document]
Comment (vigejolla) [reply sdk-setup/systemd/host_home.service:10]: Yeah, we definitely should do that at some point

**Review by vigejolla — APPROVED** [review]

**Review by vigejolla — APPROVED** [review]
lgtm

**Pull request merged**